tests/c_languages/CPPLintBearTest.py

```diff
@@ -7,6 +7,13 @@
 }
 """
 
+test_file2 = """
+int main() {
+    int x = 3, y = 3, z = 2, a = 3, b = 5, d = 3, t = 3, u = 3, j = 3;
```

Comment:
I meant to generate a very long line dynamically, but still to have valid code 😅